src/sqlfluff/core/templaters/slicers/tracer.py

```diff
@@ -215,9 +247,9 @@ def _slice_template(self) -> List[RawFileSlice]:
         # https://jinja.palletsprojects.com/en/2.11.x/api/#jinja2.Environment.lex
         stack = []
         result = []
-        set_idx = None
```

Comment:
This moved and was renamed as a new instance field on `JinjaTracer`: `inside_set_or_macro`.